packages/edit-post/src/components/header/more-menu/index.js

```diff
@@ -14,6 +14,9 @@ import ToolsMoreMenuGroup from '../tools-more-menu-group';
 import OptionsMenuItem from '../options-menu-item';
 import WritingMenu from '../writing-menu';
 
+const ariaClosed = _x( 'Show additional settings', 'button to expand options' );
```

Comment:
There's no need for adding a context here since both strings are only used once.

Reply:
```suggestion
const ariaClosed = _x( 'Show additional settings');
```